checkstyle.xml

```diff
@@ -145,7 +145,7 @@ page at http://checkstyle.sourceforge.net/config.html -->
 
     <module name="LineLength">
       <!-- Checks if a line is too long. -->
-      <property name="max" value="${com.puppycrawl.tools.checkstyle.checks.sizes.LineLength.max}" default="80"/>
+      <property name="max" value="${com.puppycrawl.tools.checkstyle.checks.sizes.LineLength.max}" default="120"/>
```

Comment:
I, and the general programming community, are against this, even though its sorta inconvenient sometimes. http://stackoverflow.com/questions/903754/do-you-still-limit-line-length-in-code